### PR TITLE
0.26.0 API upgrade

### PR DESCRIPTION
**Pull request by AStox**
This PR ...
1. upgrades the mappings to fit the newest 0.26.0 API standards
2. Fixes a bug in the AO rewards calculations that used the investor reward rate for a time period.

### CODE REVIEW

File: src/mappings/Coordinator.ts [hunk 1]
```diff
@@ -13,19 +13,24 @@ import { timestampToDate } from '../util/date'
...
   if (poolId != paperchainRoot) {
     let investors = loadOrCreatePoolInvestors(poolId)
-    let txHash = call.transaction.hash.toHex()
+    let txHash = tx.hash.toHex()
     let pool = Pool.load(poolId)
+    if (!pool) {
+      return
+    }
+
+    log.info(' ', [])
```
Comment (offerijns) [inline src/mappings/Coordinator.ts:26]:
```suggestion
```

File: subgraph-mainnet-production.yaml [hunk 2]
```diff
@@ -1,5 +1,10 @@
 specVersion: 0.0.2
 description: Tinlake Mainnet Production
+# graft:
+# base: QmNXmoKc82AHzMupxisVCAn1PHHMazqAdjjWcHyXm8m3Bj
+# block: 14044599
+# base: QmYhKDTf285fm37Gq466wWt9NJHC117YLMAi91tLo4ZhXQ
+# block: 11738159
```
Comment (offerijns) [inline subgraph-mainnet-production.yaml:7]:
```suggestion
```

File: src/mappings/Tranche.ts [hunk 3]
```diff
@@ -141,7 +153,7 @@ export function handleRedeemOrder(call: RedeemOrderCall): void {
   investorTx.currencyAmount = tokenPrice.gt(BigInt.fromI32(0))
     ? tb.pendingRedeemToken.times(tokenPrice).div(fixed27)
     : tb.pendingRedeemToken
-  investorTx.gasUsed = call.transaction.gasUsed
+  investorTx.gasUsed = call.transaction.gasLimit
```
Comment (offerijns) [inline src/mappings/Tranche.ts:156]: This is surprising. Why are we using `gasLimit` rather than ` gasUsed` here?

**Review by offerijns — APPROVED** [review]
Awesome work, a few tiny comments, but LGTM!

File: src/mappings/Tranche.ts (hunk 3 above)
Comment (AStox) [reply src/mappings/Tranche.ts:156]: This is a strange update they made in version 0.0.5 of the Mapping API:
image

**Pull request merged**